lib/src/connector/connection.cc

```diff
@@ -343,7 +343,7 @@ class verbose_verification {
         X509_NAME_oneline(X509_get_subject_name(cert), subject_name, 256);
         X509_NAME_oneline(X509_get_issuer_name(cert), issuer_name, 256);
         bool verified = verifier_(preverified, ctx);
-        LOG_TRACE("Verifying %1%, issued by %2%. Verified: %3%",
+        LOG_DEBUG("Verifying %1%, issued by %2%. Verified: %3%",
```

Comment:
Agreed, indicating verification status with an integer is confusing as heck - especially because 0 could indicate no errors, or 0 could indicate that verification failed and 1 indicates verification success. It seems more appropriate to log errors at LOG_ERROR when it fails, and then probably fail as hard as possible. SSL identity verification failure should be an irrecoverable failure.